test/e2e/v1beta1/scripts/gh-actions/run-e2e-experiment.py

```diff
@@ -216,6 +215,12 @@ def run_e2e_experiment(
         required=True,
         help="Path to the Katib Experiment.",
     )
+    parser.add_argument(
+        "--namespace", type=str, required=True, help="Namespace for the Katib E2E test",
+    )
+    parser.add_argument(
+        "--trial-pod-annotations", type=str, help="Annotation for the pod created by trial",
```

Comment:
In which formant we are going to pass annotations ?

Reply:
I saw your comment above: https://github.com/kubeflow/katib/pull/2138#discussion_r1155243034.
@nagar-ajay Did you test it if you can pass multiple annotation in that format ? E.g.:
```
'{"sidecar.istio.io/inject": "false", "custom-key": "custom-value"}' 
```

Reply:
yes